flow360/__init__.py

```diff
@@ -55,7 +56,6 @@
     PorousMedium,
     ProbeMonitor,
     SliceOutput,
-    Slices,
```

Comment:
why remove slices here?

Reply:
because Slices is imported from flow360_output.py: https://github.com/flexcompute/Flow360/pull/121/files#diff-6f2dbf2174cca53694115555325631797a3b0f630a967bf069593bb7468177f8R32